dashboard/app/controllers/api/v1/pd/workshops_controller.rb

```diff
@@ -27,6 +27,10 @@ def index
       @workshops = ::Pd::Workshop.where(id: params[:workshop_id])
     end
 
+    if params[:exclude_summer]
+      @workshops = @workshops.to_a.reject! {|w| w.local_summer? || w.teachercon?}
```

Comment:
s/reject!/reject

The bang version modifies the array directly, and returns `nil` if there are no changes. See https://apidock.com/ruby/Array/reject%21

Though, like I said above, I'd prefer this as a scope instead of using reject at all.

Reply:
moved to scope